### DISCUSSION TITLE
Welcome to tinyfilemanager Discussions!

### DISCUSSION THREAD

**Discussion by prasathmani** [prasathmani/tinyfilemanager#470] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by almusx** [comment]
Is there a way to download the file instead of showing the preview window when clicked on file name?

**Reply by prasathmani** [reply]
try this
![image](https://user-images.githubusercontent.com/11345598/116961222-67329a80-acc0-11eb-8a4e-ac9a1f445132.png)

**Reply by almusx** [reply]
Hi, thank you but i've found a simple way to do it:

Changed:

$filelink = '?p=' . urlencode(FM_PATH) . '&amp;**view**=' . urlencode($f);

to:

$filelink = '?p=' . urlencode(FM_PATH) . '&amp;**dl**=' . urlencode($f);
